docs/user/01-manager.md

```diff
@@ -11,6 +11,12 @@ The Telemetry module includes Telemetry Manager, a Kubernetes [operator](https:/
 
 ![Manager](assets/manager-lifecycle.drawio.svg)
```

Comment:
Consistency: In the diagram, either use 3rd person ("watches", "manages", "syncs") or infinitive ("read"), but don't mix them.